src/Router.js

```diff
@@ -3,14 +3,15 @@ import { BrowserRouter, Routes, Route } from 'react-router-dom';
 import ProductList from './pages/ProductList/ProductList';
 import ProductDetail from './pages/ProductDetail/ProductDetail';
 import Login from './pages/Login/Login';
-
+import Cart from '../src/pages/Cart/Cart';
 const Router = () => {
```

Comment:
- 사이에 개행 해주시면 좋을 것 같습니다-!
```suggestion
import Cart from '../src/pages/Cart/Cart';

const Router = () => {
```

Reply:
넵!!